storybook/src/pages/amopis/common/amopis.css

```diff
@@ -0,0 +1,62 @@
+.ams-docs-amopis-app-grid {
+  display: grid;
+  grid-template-columns: 112px 1fr;
+}
+
+.ams-docs-amopis-app-content {
+  display: flex;
+  flex-direction: column;
+}
+
+.ams-docs-amopis-main-background {
+  background-color: var(--ams-color-neutral-grey1);
+  flex: auto;
+}
+
+.ams-docs-amopis-white-background {
+  background-color: var(--ams-color-primary-white);
```

Comment:
We should not use primitive tokens like this but I understand it's for the demo. We've discussed a document level stylesheet. Perhaps this is a good moment to implement this, and then you can have a `document.compact.tokens.json` for Amopis. Just a thought.